rmb-sdk-go/peer/encoder/json.go

```diff
@@ -0,0 +1,21 @@
+package encoder
+
+import "encoding/json"
+
+type jsonEncoder struct{}
+
+func (e *jsonEncoder) Schema() string {
+	return JSONSchema
+}
+
+func (e *jsonEncoder) Encode(v interface{}) ([]byte, error) {
+	return json.Marshal(v)
+}
+
+func (e *jsonEncoder) Decode(data []byte, out interface{}) error {
+	return json.Unmarshal(data, out)
+}
+
+func newJSONEncoder() Encoder {
```

Comment:
This should be public IMHO, even if it's the default the user should still explictely pass it as an option